default/python/universe_generation/monsters.py

```diff
@@ -22,7 +96,7 @@ def generate_monsters(monster_freq, systems):
 
     # get all monster fleets that have a spawn rate and limit both > 0 and at least one monster ship design in it
     # (a monster fleet with no monsters in it is pointless) and store them in a list
-    fleet_plans = fo.load_monster_fleet_plan_list()
+    fleet_plans = [fp for fp in fo.load_monster_fleet_plan_list()]
```

Comment:
Sorry.  Filtering was added.  Filtering was removed.  Original line was not restored.

Original line is now restored.